CUETools/CUETools.TestCodecs/FlakeWriterTest.cs

```diff
@@ -79,24 +79,24 @@ public void ConstructorTest()
             AudioBuffer buff = Codecs.WAV.AudioDecoder.ReadAllSamples(new Codecs.WAV.DecoderSettings(), "test.wav");
             AudioEncoder target;
 
-            target = new AudioEncoder(new EncoderSettings() { PCM = buff.PCM, EncoderMode = "7" }, "flakewriter0.flac");
+            target = new AudioEncoder(new EncoderSettings() { PCM = buff.PCM, EncoderMode = "7" }, "Flakewriter0.flac");
             target.Settings.Padding = 1;
             target.DoSeekTable = false;
             //target.Vendor = "CUETools";
             //target.CreationTime = DateTime.Parse("15 Aug 1976");
             target.FinalSampleCount = buff.Length;
             target.Write(buff);
             target.Close();
-            CollectionAssert.AreEqual(File.ReadAllBytes("flake.flac"), File.ReadAllBytes("flakewriter0.flac"), "flakewriter0.flac doesn't match.");
+            CollectionAssert.AreEqual(File.ReadAllBytes("Flake.flac"), File.ReadAllBytes("Flakewriter0.flac"), "Flakewriter0.flac doesn't match.");
 
-            target = new AudioEncoder(new EncoderSettings() { PCM = buff.PCM, EncoderMode = "7" }, "flakewriter1.flac");
+            target = new AudioEncoder(new EncoderSettings() { PCM = buff.PCM, EncoderMode = "7" }, "Flakewriter1.flac");
             target.Settings.Padding = 1;
             target.DoSeekTable = false;
             //target.Vendor = "CUETools";
             //target.CreationTime = DateTime.Parse("15 Aug 1976");
             target.Write(buff);
             target.Close();
-            CollectionAssert.AreEqual(File.ReadAllBytes("flake.flac"), File.ReadAllBytes("flakewriter1.flac"), "flakewriter1.flac doesn't match.");
+            CollectionAssert.AreEqual(File.ReadAllBytes("Flake.flac"), File.ReadAllBytes("Flakewriter1.flac"), "Flakewriter1.flac doesn't match.");

```

Comment:
No need to change the file names of `flake.flac` and `flakewriter1.flac` in the `FlakeWriterTest`. They should stay lower case, in order to avoid unnecessary changes, which are not related to the issue.